lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -8210,7 +8179,20 @@ namespace Js
 
             if (length > 1)
             {
-                JavascriptString* separator = GetLocaleSeparator(scriptContext);
+                char16 szSeparator[6];
+                // According to the document for GetLocaleInfo this is a sufficient buffer size.
```

Comment:
Which documentation?

Reply:
This comment is currently in JavascriptArray::GetLocaleSeparator. Because I moved the buffer outside the function I didn't want to leave its size a magic number.

Reply:
I see. Moving that documentation to this new PlatformAgnostic class seems useful, as would making this magic number a static constant if its going to be used.